docs/src/pages/docs/crocks/Async.md

```diff
@@ -380,8 +379,8 @@ As such, the need for binding may arise. `fromNode` provides a second, optional
 argument that takes the context that will be used to bind the function being
 wrapped.
 
-Like [`fromPromise`](#frompromise), any curried interface will not be respected.
-If a curried interface is needed then [`nAry`][nary] can be used.
+Any curried interface will not be respected and if a curried interface is needed 
```

Comment:
Good 👀!
This makes me wonder if there is a similar change in `fromNode`'s near future

Reply:
Might be worth a quick look around the library to see if any of these `from` could use the change and an issue could be created to track it